app/views/support_interface/regions/edit.html.erb

```diff
@@ -5,29 +5,30 @@
 <%= form_with model: [:support_interface, @region] do |f| %>
   <%= f.govuk_error_summary %>
 
+  <%= render "shared/support_interface/forms/application_form", f: %>
```

Comment:
This field `eligibility_enabled` doesn't exist on regions so we can't edit it here.